guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -31,7 +31,8 @@ Use this procedure to enable the repositories that are required to install {Prod
 +
 [NOTE]
 ====
-include::snip_satellite-el8-enable-note.adoc[]
+:dnf-module: satellite-capsule:el8
```

Comment:
Perhaps this should be set a bit higher and also used on line 29 so you know they're connected?